### PR TITLE
libpython: explain verbosity levels

### PR DESCRIPTION
**Pull request by neteler**
added missing docs, cited from https://github.com/OSGeo/grass/blob/8b9512b8b2fb0531df848fce58619308cbc29b3d/lib/gis/verbose.c#L46

### CODE REVIEW

File: lib/python/script/core.py [hunk 1]
```diff
@@ -1510,7 +1510,15 @@ def overwrite():
 
 
 def verbosity():
-    """Return the verbosity level selected by GRASS_VERBOSE"""
+    """Return the verbosity level selected by GRASS_VERBOSE
+
+    Currently, there are 5 levels of verbosity:
+    -1 nothing will be printed (also fatal errors and warnings will be also discarded)
```
Comment (petrasovaa) [inline lib/python/script/core.py:1516]:
```suggestion
    -1 nothing will be printed (also fatal errors and warnings will be discarded)
```

File: lib/python/script/core.py [hunk 2]
```diff
@@ -1510,7 +1510,15 @@ def overwrite():
...
+     0 nothing will be printed except of fatal errors and warnings. Triggered by --q or --quiet flag.
```
Comment (petrasovaa) [inline lib/python/script/core.py:1517]:
```suggestion
     0 only errors and warnings are printed, triggered by --q or --quiet flag.
```

File: lib/python/script/core.py [hunk 3]
```diff
@@ -1510,7 +1510,15 @@ def overwrite():
...
+     1 only progress information (percent) and important messages will be printed
```
Comment (petrasovaa) [inline lib/python/script/core.py:1518]:
```suggestion
     1 progress information (percent) and important messages will be printed
```

**Review by petrasovaa — APPROVED** [review]
Some suggestions to simplify wording.

**Comment by neteler** [comment]
note: layout improved in #1004

**Pull request merged**